content/features/service_discovery.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Service Discovery"
+file: "service_discovery"
+date: 2018-07-18T12:33:50+02:00
+draft: false
+youTubeID: oG-WDA-hmvA
+subTitle: "Service view, Pods, Deployments, Rules, Metrics..."
```

Comment:
The "..." sounds weird here.
What about ?
"Services view focus on Istio observability"

Just to show something generic, for me "Pods/Deployments/Rules/Metrics" is more a detail it can go below.

Reply:
Right